pkg/cmd/main_test.go

```diff
@@ -31,7 +32,10 @@ func ExecuteCommand(args string) (string, string) {
 	cmd.SetOut(&stdout)
 	cmd.SetErr(&stderr)
 	cmd.SetArgs(parseArgs(args))
-	cmd.Execute()
+	err := cmd.Execute()
+	if err != nil {
+		return stdout.String(), err.Error()
```

Comment:
We should actually change the signature of `ExecuteCommand` to return `(string, string, error)` instead of `(string, string)`. I don't think returning `err.Error()` in place of the stderr here is actually correct.